### PR TITLE
Fix twitter cards

### PR DESCRIPTION
**Pull request by maraoz**

### CODE REVIEW

**Comment by maraoz** [comment]
It works! You can go to: https://cards-dev.twitter.com/validator
and enter the url: https://deploy-preview-105--zeppelinsolutions.netlify.com/jobs/frontend-engineer/ to check it out.

File: _includes/head.html [hunk 1]
```diff
@@ -3,41 +3,43 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   
+  <meta name="twitter:description" content="View the album on Flickr." />
+  <meta name="twitter:image" content="https://farm6.staticflickr.com/5510/14338202952_93595258ff_z.jpg" />
```
Comment (frangio) [inline _includes/head.html]: These two lines seem to be left over from testing. :slightly_smiling_face:

File: _includes/head.html [hunk 2]
```diff
@@ -3,41 +3,43 @@
...
-  <link rel="apple-touch-icon" sizes="120x120" href="{{ "/images/favicon/apple-icon-120x120.png" | absolute_url }}"/>
-  <link rel="apple-touch-icon" sizes="144x144" href="{{ "/images/favicon/apple-icon-144x144.png" | absolute_url }}"/>
-  <link rel="apple-touch-icon" sizes="152x152" href="{{ "/images/favicon/apple-icon-152x152.png" | absolute_url }}"/>
-  <link rel="apple-touch-icon" sizes="180x180" href="{{ "/images/favicon/apple-icon-180x180.png" | absolute_url }}"/>
-  <link rel="icon" type="image/png" sizes="192x192"  href="{{ "/images/favicon/android-icon-192x192.png" | absolute_url }}"/>
-  <link rel="icon" type="image/png" sizes="32x32" href="{{ "/images/favicon/favicon-32x32.png" | absolute_url }}"/>
-  <link rel="icon" type="image/png" sizes="96x96" href="{{ "/images/favicon/favicon-96x96.png" | absolute_url }}"/>
-  <link rel="icon" type="image/png" sizes="16x16" href="{{ "/images/favicon/favicon-16x16.png" | absolute_url }}"/>
-  <link rel="manifest" href="{{ "/images/favicon/manifest.json" | absolute_url }}"/>
+  <link rel="apple-touch-icon" sizes="57x57" href="{{ "/images/favicon/apple-icon-57x57.png" }}"/>
```
Comment (frangio) [inline _includes/head.html]: Nitpick: since these are static strings now the `{{ "` `" }}` are not necessary.

**Comment by maraoz** [comment]
Thanks for the review @frangio , should be fixed now

File: _includes/head.html [hunk 3]
```diff
@@ -6,38 +6,38 @@
...
-  <link rel="apple-touch-icon" sizes="180x180" href="{{ "/images/favicon/apple-icon-180x180.png" | absolute_url }}"/>
-  <link rel="icon" type="image/png" sizes="192x192"  href="{{ "/images/favicon/android-icon-192x192.png" | absolute_url }}"/>
-  <link rel="icon" type="image/png" sizes="32x32" href="{{ "/images/favicon/favicon-32x32.png" | absolute_url }}"/>
-  <link rel="icon" type="image/png" sizes="96x96" href="{{ "/images/favicon/favicon-96x96.png" | absolute_url }}"/>
-  <link rel="icon" type="image/png" sizes="16x16" href="{{ "/images/favicon/favicon-16x16.png" | absolute_url }}"/>
-  <link rel="manifest" href="{{ "/images/favicon/manifest.json" | absolute_url }}"/>
+  <link rel="apple-touch-icon" sizes="57x57" href="/images/favicon/apple-icon-57x57.png"/>
+  <link rel="apple-touch-icon" sizes="60x60" href="/images/favicon/apple-icon-60x60.png"/>
+  <link rel="apple-touch-icon" sizes="72x72" href="/images/favicon/apple-icon-72x72.png"/>
+  <link rel="apple-touch-icon" sizes="76x76" href="/images/favicon/apple-icon-76x76.png/">
```
Comment (frangio) [inline _includes/head.html]: Accidentally switched characters.
```suggestion
  <link rel="apple-touch-icon" sizes="76x76" href="/images/favicon/apple-icon-76x76.png"/>
```

**Pull request merged**